Guide.docc/DataRaceSafety.md

```diff
@@ -226,7 +225,7 @@ but that doesn't mean you must always manually start one.
 Typically, asynchronous functions do not need to be aware of the
 task running them.
 In fact, tasks can often begin at a much higher level,
-within an application framework, or even at the root of a program.
+within an application framework, or even at the entry point of the program.
```

Comment:
what is "the root of a program"?

Reply:
hehe yeah... well, not where the leafs are! 🥬 

Good rewording